packages/fx-core/src/plugins/resource/sql/managementClient.ts

```diff
@@ -217,11 +217,12 @@ export class ManagementClient {
       startIpAddress: startIp,
       endIpAddress: endIp,
     };
+    const ruleName = Constants.firewall.localRule + index;
```

Comment:
Can we update the rule instead of add multiple rules? Since the api is createOrUpdate. What is the reason to add multiple rules?

Reply:
The old firewall rule is kept as the local IP may change in 2 address back and forth. It could successfully access SQL if the IP change back to the old one as well.